File: CashLogApi/CashLogUtils/Validators.cs
using System;

namespace CashLogUtils
{
    class Validators
    {
        Helpers helper = new Helpers();
        public bool ValidateCpf(string value)
        {
            if (value == null || value.Length != 11 || !Decimal.TryParse(value, out decimal result)) return false;
            return true;
        }

        public bool ValidateEmail(string value)
        {
            if (value == null || !value.Contains("@") || !value.Contains(".")) return false;
            return true;
        }

        public bool ValidateCnpj(string value)
        {
            if (value == null || value.Length != 14 || !Decimal.TryParse(value, out decimal result)) return false;
            return true;
        }

        public bool ValidateCep(string value)
        {
            if (value == null || value.Length != 8 || !helper.IsNumber(value)) return false;
            return true;
        }

    }
}

File: CashLogApi_VBnet/LazyDevUtils/Validators.vb
Imports LazyDevUtils.Extensions

Namespace Validators

    Public Class Validators

        Sub New()
        End Sub

        Public Function ValidateCpf(value As String) As Boolean
            If value Is Nothing OrElse value.Length <> 11 OrElse Not Decimal.TryParse(value, Nothing) Then Return False
            Return True
        End Function

        Public Function ValidateEmail(value As String) As Boolean
            If value Is Nothing OrElse Not value.Contains("@") OrElse Not value.Contains(".") Then Return False
            Return True
        End Function
        Public Function ValidateCnpj(value As String) As Boolean
            If value Is Nothing OrElse value.Length <> 14 OrElse Not Decimal.TryParse(value, Nothing) Then Return False
            Return True
        End Function
        Public Function ValidateCep(value As String) As Boolean
            If value Is Nothing OrElse value.Length <> 8 OrElse Not IsNumber(value) Then Return False
            Return True
        End Function

    End Class

End Namespace